css/index.css

```diff
@@ -1,3 +1,278 @@
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
 
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Great Idea CSS */
+
+.sign {
+	font-size: 80px;
+	font-family:'Bangers', cursive;
+	text-align: center;
+	margin-top:100px;
+	padding:0px 100px 0px 0px;
+}
+
+ header {
+ 	display: flex;
+	align-items:center;
+ }
+ 
+ header nav a{
+	padding:5px 20px;
+	background: white;
+	margin:10px;
+	color:gray;
+	text-decoration: none;
+}
+
+.container {
+	margin:0 auto;
+	width: 900px;
+}
+
+.top-section {
+	display:flex;
+	justify-content: flex-start;
+}
+
+.top-section .top-paragraph {
+	width:450px;
+	margin-top:20px;
+	margin-bottom: 25px;
+
+} 
+
+.button {
+	border:1px solid black;
+	background:white;
+	height:40px;
+	width:200px;
+	box-sizing: border-box;
+	font-size: 20px;
+	} 
+
+
+.circle {
+	margin-top: 50px;
+} 
+
+.sub-head {
+	display:flex;
+	justify-content: flex-end;
+	margin-bottom:75px;
+}
+
+.middle-img {
+    display: block;
+}
+
+.dash {
+	border:0 none;
+    border-top: 1px solid #000;
+    width:100%;
+    height:0px;
+    background:none;
+}
+
+.bottom-section {
+	display:flex;
+	justify-content: space-between;
+}
+
+.bottom-section .paragraph-content {
+	width:275px;
+	margin-top:25px;
+	margin-bottom:25px;
+}
+
+address {
+	display:flex;
+	flex-direction: column;
+	justify-content: space-around;
+}
+
+.copyright {
+	text-align:center;
+	margin: 50px;
+}
+
+.font {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+	margin-bottom: 5px;
+	margin-top:15px;
+}
+
+
+@media (max-width:800px) {
+	.container {
+		margin:0 auto;
+		width:800px;
+		
+	}
+	.circle {
+		display:none;
+	}
+	
+	.sub-head {
+		display:flex;
+		justify-content: center;
+	}
+
+	.top-section {
+		display:flex;
+		justify-content: center;
+	}
+
+	.top-section .top-paragraph {
+		width:380px;
+		margin-top:20px;
+		margin-bottom: 25px;
+	} 
+	
+	header {
+	    display: flex;
+		justify-content:center;
+		flex-direction: column-reverse;
+	}
+
+	header nav a{
+		padding:0px 40px;
+		background: white;
+		margin:0px;
+		color:gray;
+		text-decoration: none;
+	}
+
+	#logo {
+		margin: 30px;
+	}
+
+	.sub-head {
+		display:flex;
+		justify-content: center;
+	}
+
+	.sign {
+		margin-top:100px;
+        padding:0px;
+	}
+
+    .bottom-section {
+		display:flex;
+		justify-content:center;
+	}
+
+	img {
+		max-height:100%;
+		max-width:100%;
+	}
+}
+
+@media (max-width:500px) {
+	.container {
```

Comment:
nicely do by adding all the changes wanted to the media queries and keeping code looking nice